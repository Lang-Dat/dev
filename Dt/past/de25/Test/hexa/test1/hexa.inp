1110010010100011010110101110010001010000010010111111101001000110000010110111
