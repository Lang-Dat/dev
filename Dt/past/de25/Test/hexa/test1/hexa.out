E4A35AE4504BFA460B7
